deployment-aws/puppet/cromwell/manifests/install.pp

```diff
@@ -15,5 +15,9 @@
     source => "https://github.com/broadinstitute/cromwell/releases/download/${version}/cromwell-${version}.jar"
   }
 
+  file { "/opt/amazon-cloudwatch-agent.deb":
```

Comment:
There is a `package` resource for managing packages:
https://stackoverflow.com/a/58977424
https://puppet.com/docs/puppet/7/types/package.html